go/libkb/constants.go

```diff
@@ -372,3 +372,7 @@ const (
 const (
 	SecretPromptCancelDuration = 5 * time.Minute
 )
+
+const ServiceLogFileName = "keybase.service.log"
```

Comment:
We could use IsBrewBuild and then return the homebrew.mxcl. versions of these but then I guess would need to make them functions

But it is nice to only have 1 place for log file... I'm ok with it